persistence/sql/persister_identity.go

```diff
@@ -253,73 +250,65 @@ func (p *Persister) CreateIdentity(ctx context.Context, i *identity.Identity) er
 	})
 }
 
-func (p *Persister) ListIdentities(ctx context.Context, page, perPage int) ([]identity.Identity, error) {
-	is := make([]identity.Identity, 0)
-
-	/* #nosec G201 TableName is static */
-	if err := sqlcon.HandleError(p.GetConnection(ctx).Where("nid = ?", corp.ContextualizeNID(ctx, p.nid)).
-		EagerPreload("VerifiableAddresses", "RecoveryAddresses").
-		Paginate(page, perPage).Order("id DESC").
-		All(&is)); err != nil {
-		return nil, err
-	}
-
-	schemaCache := map[string]string{}
+func (p *Persister) setIdentitySchemas(ctx context.Context, is []identity.Identity) error {
```

Comment:
I extracted this into a common function because it was copy&pasted in the filter code